src/Compilers/Core/MSBuildTaskTests/IntegrationTests.cs

```diff
@@ -615,6 +622,8 @@ public void ReportAnalyzerMSBuild()
         [Fact(Skip = "failing msbuild")]
         public void SolutionWithPunctuation()
         {
+            RoslynDebug.Assert(s_msbuildExecutable is object);
```

Comment:
>s_msbuildExecutable [](start = 31, length = 19)

I'm wondering whether we could strengthen the type of `s_msbuildExecutable` to be non-nullable, by throwing from the static constructor

Reply:
I was able to strengthen it by making it an instance field.